draft-ietf-quic-tls.md

```diff
@@ -618,11 +618,18 @@ application data, a server MUST treat receiving early data on the TLS connection
 as a connection error of type PROTOCOL_VIOLATION.
 
 
-## Rejecting 0-RTT
+## Accepting and rejecting 0-RTT
 
-A server rejects 0-RTT by rejecting 0-RTT at the TLS layer.  This also prevents
-QUIC from sending 0-RTT data. A server will always reject 0-RTT if it sends a
-TLS HelloRetryRequest.
+A server accepts 0-RTT by sending the EarlyDataIndication in the ServerHello
```

Comment:
I think you might want to clarify that the client uses the early_data extension when it is going to send 0-RTT packets. That's implied by this sentence (because in TLS a server cannot send the early_data extension unless client has sent one), though.

Reply:
@kazuho I added a paragraph about that. PHAL.

Reply:
I think we need to state "the early_data extension" instead of "EarlyDataIndication".

EarlyDataIndication is type of a structure, that affects the peer only if it is being carried by the early_data extension.

Reply:
```suggestion
A server accepts 0-RTT by sending an `early_data` extension in the ServerHello
```